### PR TITLE
Blog post for LASTConf intro to wpo talk

### PR DESCRIPTION
**Pull request by tpartington**

### CODE REVIEW

File: content/post/2017-10-06-intro-to-wpo.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+tags:
+    - devops
+    - video
+author: Tom Partington
+excerpt: This is an excerpt from a talk I gave at LASTConf Melbourne 2017 on Web Performance Optimisation.
+url: devops/2017/10/06/intro_to_wpo.html
+---
+This is an excerpt from a [talk](https://confengine.com/last-conference-melbourne-2017/proposal/4034/an-introduction-to-web-performance-optimisation-practical-steps-for-reducing-costs-and-improving-the-user-experience) I gave at [LASTConf Melbourne](https://www.lastconference.com/melbourne/) on Web Performance Optimisation.  
+
+In this short I discuss some of the benefits that web optimisation can bring to internal web pages, and what this means for our concentration and stress levels.
```
Comment (stevemac007) [inline content/post/2017-10-06-intro-to-wpo.md]: Short video?

Comment (mngau) [reply content/post/2017-10-06-intro-to-wpo.md]: I like excerpt.

Comment (mngau) [reply content/post/2017-10-06-intro-to-wpo.md]: Or maybe add 2min there?

Comment (mngau) [reply content/post/2017-10-06-intro-to-wpo.md]: e.g. short 2 min video

**Pull request merged**